trollmoves/tests/test_client.py

```diff
@@ -167,6 +167,17 @@
 nameservers = ns1 ns2
 """
 
+CLIENT_CONFIG_BACKUP_TARGETS = """
+[foo]
+providers = bar
+destination = scp://primary_host/tmp/foo
+login = user
+topic = /1b/hrit-segment/0deg
+publish_port = 0
+nameservers = ns1
+backup_targets=backup_host1 backup_host2
```

Comment:
These hosts need to have the same protocol as the original destination.

Reply:
Yeah, and that reminds me of adding an example explaining this.